### PR TITLE
message updated chatbot

### PR DESCRIPTION
**Pull request by PhodzoW0l7**
Integrating the chatbot to my previous personal portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -383,7 +383,7 @@ <h2>Chatbot</h2>
   <ul class="chatbox">
     <li class="chat incoming">
       <span class="material-symbols-outlined">smart_toy</span>
-      <p>Hi there <br>How can I help you today?</p>
+      <p>Hi,<br>How can I help you today?</p>
```
Comment (Tepanyega) [inline index.html]: @PhodzoW0l7 
i think it will be better if you add "there" hahahahaha, like 
"hi, there"

**Review by Tepanyega — CHANGES_REQUESTED** [review]

File: index.html
Comment (alphineandre) [inline index.html:1]: The files look incomplete. the CSS and JS files are also not here so we are not able to review the chatbot fully

**Review by alphineandre — CHANGES_REQUESTED** [review]
Fix the files so we can review everything.

**Review by bokangkhumalo — APPROVED** [review]
Hi @PhodzoW0l7, Website is looking inviting and user friendly, i like the use of a bright color pallete and simple animations that give the user feedback.

File: index.html (hunk 1 above)
Comment (PhodzoW0l7) [reply index.html]: Noted.

**Review by KhanyiCodes — DISMISSED** [review]

File: index.html
Comment (KhanyiCodes) [reply index.html:1]: True, we can only see the HTML file.

Comment (bokangkhumalo) [reply index.html:1]: @PhodzoW0l7, Agreed, im just here to support this review as i had noticed this at a later stage, please make sure this is updated for your next review before approval. Happy coding!

**Review by bokangkhumalo — CHANGES_REQUESTED** [review]
Hi, @PhodzoW0l7, please Ensure that all files are included in your files changed before you can merge.

**Review by Tepanyega — APPROVED** [review]

File: style/chatstyle.css
Comment (PhodzoW0l7) [inline style/chatstyle.css:1]: Removed the styling file and added the styling to them main CSS which is style.css for better integration.

File: javascript/chatscript.js
Comment (PhodzoW0l7) [inline javascript/chatscript.js:1]: @alphineandre @bokangkhumalo @KhanyiCodes  I added JavaScript information to feed my data with responses and works perfectly.

File: style/style.css
Comment (PhodzoW0l7) [inline style/style.css:1]: matching the colors for better visuals when user interacts with the chatbot.

**Review by JKRakuba — CHANGES_REQUESTED** [review]
Hey @PhodzoW0l7  I've noticed that the responsiveness of the website isn't showing the full navigation bar for small screens. Please also look into that

File: chatbot/main.js
Comment (alphineandre) [inline chatbot/main.js:1]: Okay leaving no stone unturned i see.

File: chatbot/index.html
Comment (alphineandre) [inline chatbot/index.html:1]: Looks good

**Review by alphineandre — APPROVED** [review]
Everything looks good. Great work

**Review by bokangkhumalo — APPROVED** [review]
Awesome work @PhodzoW0l7, nice to see some advanced features implemented!

**Review by JKRakuba — APPROVED** [review]
Well done!

**Review by KhanyiCodes — APPROVED** [review]

**Pull request merged**